prism/src/main/java/org/apache/falcon/resource/AbstractInstanceManager.java

```diff
@@ -676,6 +683,79 @@ public TriageResult triageInstance(String entityType, String entityName, String
         }
     }
 
+    public InstancesResult searchInstances(String type, String nameSubsequence, String tagKeywords,
```

Comment:
Could you elaborate your thoughts? This function retrieves relevant entities and then make a batch query on their instances on Titan DB.